content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -315,6 +315,11 @@ $ ORIGINAL_SVC_MANIFEST=mysvc-v1.yaml
 $ istioctl kube-inject --injectConfigMapName istio-sidecar-injector --meshConfigMapName istio -f ${ORIGINAL_SVC_MANIFEST} | kubectl apply -f -
 {{< /text >}}
 
+## Accessing Services from different clusters
+
+To access a service, the client first needs to resolve the service DNS name to an IP address. Once the name is resolved, Istio will handle routing to service endpoints in any cluster.
```

Comment:
Line length and passive voice issues: These apply everywhere in the document but I will only mention them once. The acceptable line length is 80 characters. Avoid passive voice. Specifically here, you should avoid future tense too.  It creates a grammatical issue by breaking verb tense congruence. Remember to add articles before nouns.
I suggest:
Once the client resolves the name, Istio handles the routing to the service endpoints in any cluster.